### ISSUE TITLE
Incorrect printing of record types in coqide

### ISSUE THREAD

**Issue by silene** [coq/coq#8398]
#### Version

8.9

#### Description of the problem

```
Record foo := { bar : nat }.
Print foo.
```

When run in Coqide, this gives

```
Record foo : Set := Build_foo { bar : nat
For Build_foo: Argument scope is [nat_scope]
```

So, there is as missing bracket, as well as a missing line break.

**Comment by Zimmi48** [comment]
This regression was introduced in the diffs PR #6801.

**Comment by herbelin** [comment]
... and maybe fixed by #8394 ??

**Comment by Zimmi48** [comment]
Indeed :tada:

**Comment by jfehrle** [comment]
> ... and maybe fixed by #8394 ??

Yes, indeed.  #8394 should also be marked as a blocker?

![image](https://user-images.githubusercontent.com/1253341/45043021-1ca57f80-b03a-11e8-9327-6e5b7848c8dd.png)
